src/components/molecules/CategoryEdit/index.vue

```diff
@@ -0,0 +1,145 @@
+<template>
+  <div class="category-edit">
+    <div class="category-edit__columns">
+      <section class="category-edit-editor">
+        <app-heading :level="1">カテゴリーの更新</app-heading>
+        <div class="category-edit__back">
+          <app-router-link
+            block
+            underline
+            key-color
+            hover-opacity
+            to="/categories"
+          >
+            カテゴリー一覧へ戻る
+          </app-router-link>
+        </div>
+        <div class="category-edit-form">
+          <app-input
+            v-validate="'required'"
+            name="title"
+            type="text"
+            placeholder="カテゴリーのタイトルを入力してください。"
+            white-bg
+            data-vv-as="カテゴリーのタイトル"
+            :value="categoryTitle"
+            :error-messages="errors.collect('title')"
+            @update-value="$emit('edited-title', $event)"
+          />
+        </div>
+        <app-button
+          class="category-edit-submit"
+          button-type="submit"
+          round
+          :disabled="!disabled"
+          @click="handleSubmit"
+        >
+          {{ buttonText }}
+        </app-button>
+      </section>
+    </div>
+    <div v-if="errorMessage" class="category-edit__notice">
+      <app-text bg-error>{{ errorMessage }}</app-text>
+    </div>
+    <div v-if="doneMessage" class="category-edit__notice">
+      <app-text bg-success>{{ doneMessage }}</app-text>
+    </div>
+  </div>
+</template>
+
+<script>
+import {
+  Heading, Input, Button, Text, RouterLink,
+} from '@Components/atoms';
+
+export default {
+  components: {
+    appHeading: Heading,
+    appInput: Input,
+    appButton: Button,
+    appText: Text,
+    appRouterLink: RouterLink,
+  },
+  props: {
+    categoryId: {
+      type: Number,
+      default: 0,
+    },
+    categoryTitle: {
+      type: String,
+      default: '',
+    },
+    categoryList: {
+      type: Array,
+      default: () => [],
+    },
+    categories: {
+      type: Array,
+      default() {
+        return [];
+      },
+    },
+    loading: {
+      type: Boolean,
```

Comment:
[imo]

Boolean を表現する変数名は、`is〇〇`とすることが慣習的になっているかと思います。
こういった変数名やプロパティ名の命名は、他の開発者にその変数を使って何をしているのかをぱっと見でわかりやすくするということを意識して定義するべきかと思います。

```suggestion
    isLoading: {
      type: Boolean,
```

Reply:
ありがとうございます。今後は真偽値を表現する変数名は「is〇〇」で命名するように意識いたします。変更いたしました。[27a4445](https://github.com/gizumo-education/gizumo_wiki_lesson/pull/115/commits/27a44452e78e030c5fe02df20db2b93488bbcecc)